cloudferrylib/os/image/glance_image.py

```diff
@@ -190,10 +190,10 @@ def image_exists(self, image_id):
             except glance_exceptions.HTTPNotFound:
                 return False
 
-    def get_image_list(self):
+    def get_image_list(self, is_filtering=True):
         images = self.glance_client.images.list(filters={"is_public": None})
```

Comment:
"Considered harmful" in some cases. IMHO, this is not the case.
https://blogs.msdn.microsoft.com/oldnewthing/20060828-18/?p=29953 
But I agree with the fact that the use of filtering methods like *_by_id is meaningless and can be removed.